### PR TITLE
Beta3 Release Updates (Plus some tagging of blog posts)

### PR DESCRIPTION
**Pull request by jt15s**
Added tags to some of the blog posts related to the Haiku Promotion Team, and also added WIP release pages for Beta 3 (based on the ones for Beta 2) - these have been marked as "draft=true" so Hugo will not publish these. Feel free to add any info/corrections!

Edit: Also forgot to mention that I've copied the links table under the "Getting Involved" section on the "Community" page into the "Getting Involved" page - we've been linking to the "Getting Involved" page a lot so we should have the table present on the "Getting Involved" page as it lists some of the main ways people can get involved.

### CODE REVIEW

File: content/get-haiku/r1beta3/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## New Contributors
+
+## Source code
+
+The source code of Haiku itself can be accessed from GitHub at <https://github.com/haiku/haiku> or via Haiku's Git instance at <https://git.haiku-os.org>.
+
+## Reporting issues
+
+There are almost 3200 open tickets on Haiku's bug tracker and over 12700 closed items.  If you find what you believe to be an issue, please search the bug tracker to see if it has already been reported, and if not, file a new ticket at <https://dev.haiku-os.org/>.
```
Comment (nielx) [inline content/get-haiku/r1beta3/release-notes.md:53]: 3500 open tickets 13300 closed

File: content/get-haiku/r1beta3/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
++++
+type = "article"
+title = "R1/beta3 – Release Notes"
+tags = []
+date = "2020-06-09 00:00:00Z"
+draft = "true"
++++
+**WIP Draft - this has not been finalised!**
+
+The third beta for Haiku R1 marks twenty months of hard work to improve Haiku's hardware support and its overall stability. Since Beta 2, there have been 101 contributors with over 2800 code commits in total. More than 900 bugs and enhancement tickets have been resolved for this release.
```
Comment (nielx) [inline content/get-haiku/r1beta3/release-notes.md:10]: Stats need to be updated

**Review by nielx — COMMENTED** [review]
Some improvements for the beta 3 page

**Comment by korli** [comment]
It would be nice to see the change preview like in https://github.com/haiku/website/pull/487#issuecomment-811606438

**Comment by jt15s** [comment]
> It would be nice to see the change preview like in [#487 (comment)](https://github.com/haiku/website/pull/487#issuecomment-811606438)

Unfortunately, I have no idea how to do that - and I'm not even sure if we need to set anything up etc. haha

**Comment by leavengood** [comment]
For everyone's information, that branch preview is a feature of Netlify. But for security reasons we have not set it up to work for Pull Requests from someone else's fork, because the build script is just a shell script and in theory, someone could change that and do whatever they want on Netlify's build servers. This risk is maybe overblown though.

My PR had a preview because it was just a branch on this repo, not in a fork.

Maybe we can just add jt15s to this project and he can just make changes in a branch here, and then we can get previews. Or maybe his fork can be made "safe" or something in Netlify. Unfortunately I don't think you can ask for deployment previews on an ad hoc basis.

**Comment by Coldfirex** [comment]
Adding a couple page updates from Bug tracker for the beta 3 release.
[https://dev.haiku-os.org/ticket/16676](url)
[https://dev.haiku-os.org/ticket/16988](url)
[https://dev.haiku-os.org/ticket/16983](url)

File: content/guides/building/port_status.html [hunk 3]
```diff
@@ -51,7 +57,16 @@
 	<td><img src='/files/status-icons/no-work.png' alt='No Work Complete'></td>
 	<td><img src='/files/status-icons/no-work.png' alt='No Work Complete'></td>
 	<td>0%</td></tr>
+</table>
 
+<table>
+<tr><td COLSPAN=7><h3>RISC-V (Tier 2)</h3>An open-source computer architecture with many implementations. **Note that real RISC-V hardware is not yet supported. There are issues with some packages and libstdc++ is partially broken. Additionally, nightly builds are not yet available.**</td></tr>
+<tr><th>Platform</th><th>Interest</th><th>Target</th><th>Haiku Loader</th><th>Haiku Kernel</th><th>Application Server</th><th>Status</th></tr>
+<tr><td>TinyEMU (raw platform)</td><td>High</td><td>x86_gcc2</td><td><img src='/files/status-icons/complete.png' alt='Complete'></td><td><img src='/files/status-icons/complete.png' alt='Complete'></td><td><img src='/files/status-icons/complete.png' alt='Complete'></td><td>Under Development</td></tr>
```
Comment (X547) [inline content/guides/building/port_status.html:65]: Why use x86_gcc2, x86 as targets? Both TinyEMU and QEMU are cross platform and can run on many host architectures and operating systems (32 and 64 bit x86, Haiku and openSUSE Linux are confirmed). Haiku internal target name for RISC-V is riscv64.

Comment (jt15s) [reply content/guides/building/port_status.html:65]: @X547 Sorry, I forgot to edit that part of the table! I'll fix it now.

**Comment by leavengood** [comment]
I realize we can't even preview these in Netlify since they are draft anyhow. I think we should merge this, especially since it has other changes which are unrelated to beta 3. The drafts related to beta 3 can be then be updated by anyone when we release it.

@jt15s You are happy with this beyond all the stuff we know is WIP?

**Comment by jt15s** [comment]
@leavengood I had a look at the changes I made, and I added the updated Beta3 graphic to the site, so if we merge now, there'll be a ton of confused site visitors. 

I'll remove the graphic with the current Beta2 one and then we can merge. I'll mention you when that happens.

Thanks!

**Comment by jt15s** [comment]
@leavengood Should be good to merge now, thanks!

**Comment by leavengood** [comment]
I had to fix a merge issue in the ports status page, maybe give that a look after merge.

**Pull request merged**